src/cmake/SetupBLT.cmake

```diff
@@ -23,6 +23,10 @@ if(NOT ENABLE_FOLDERS)
     set(ENABLE_FOLDERS TRUE CACHE STRING "")
 endif()
 
+################################################################
+# Make sure BLT exports its built in targets
```

Comment:
I think this should be [built-in](https://www.merriam-webster.com/dictionary/built-in) instead.

Reply:
- [ ] cyrus fix this